x-pack/docs/en/settings/security-settings.asciidoc

```diff
@@ -1225,6 +1225,13 @@ Password to the truststore.
 `xpack.ssl.truststore.secure_password` (<<secure-settings,Secure>>)::
 Password to the truststore.
 
+Note that storing trusted certificates in a PKCS#12 keystore, although supported,
+is uncommon in practice. Usually, keystore files only contain secret and private
+entries. To confirm that the passed keystore contains trusted certificate
```

Comment:
We should explicitly mention that 

> Elastic's `certutil` tool is designed to generate PKCS#12 files that can be used as both a keystore and a truststore, but this may not be the case for certificates that are generated by other tools.

Otherwise we risk confusing customers about whether our instructions work correctly with our own tools.